package.json

```diff
@@ -10,16 +10,15 @@
     "check-updates": "./scripts/check-updates",
     "clean": "rm -rf coverage; lerna exec -- rimraf lib dist",
     "commit": "git-cz",
-    "build": "lerna exec -- babel src --out-dir lib",
-    "dist": "lerna run dist",
+    "build": "lerna run --parallel build",
     "lint": "flow check && ./scripts/lint packages",
-    "test": "npm run build && jest --config test-config.json",
-    "test:package": "npm run build && ./scripts/test-package",
-    "it": "npm run it:install && npm run it:run",
-    "it:run": "jest --config integration-tests/config.json",
-    "it:install": "npm run build && ./scripts/install-npm-packages-for-integration-tests",
-    "coverage": "npm run build && jest --config test-config.json --no-watchman --coverage",
-    "coverage:ci": "jest --config test-config.json --no-watchman --coverage && codecov",
+    "test": "cross-env NODE_ENV=test jest --config .jestrc.json",
```

Comment:
do we need to set the _NODE_ENV_ using `cross-env`, we already set it in the jest config file.

Reply:
Theoretically not. However I would prefer having envs always set in the package.json and the run scripts for explicitness.